f2/src/utils/saveRecord.js

```diff
@@ -52,6 +45,13 @@ async function saveRecord(data, res) {
   }
 }
 async function getReportCount() {
+  const date = new Date()
+  const currentDate =
```

Comment:
Just threw up a comment into slack, but this is the better place.

Check out the image below as a simplified version of the code below.

![image](https://user-images.githubusercontent.com/17616596/79595057-33340980-80ac-11ea-8b6c-f25c9795bc9b.png)